website/tsconfig.json

```diff
@@ -1,11 +1,4 @@
 {
   "extends": "../tsconfig.json",
-  "compilerOptions": {
-    "baseUrl": ".",
-    "paths": {
-      "@/*": ["*"]
```

Comment:
I think this was nice :)

Reply:
I agree, if you find a way to make it pass `pnpm lint`, let's put it back!